Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -181,18 +181,17 @@ function mac_process_network_entitlements()
 function webcontent_sandbox_entitlements()
 {
     plistbuddy Add :com.apple.private.security.mutable-state-flags array
-    plistbuddy Add :com.apple.private.security.mutable-state-flags:0 string EnableMachBootstrap
+    plistbuddy Add :com.apple.private.security.mutable-state-flags:0 string AppCacheDisabled
     plistbuddy Add :com.apple.private.security.mutable-state-flags:1 string EnableExperimentalSandbox
     plistbuddy Add :com.apple.private.security.mutable-state-flags:2 string BlockIOKitInWebContentSandbox
     plistbuddy Add :com.apple.private.security.mutable-state-flags:3 string LockdownModeEnabled
     plistbuddy Add :com.apple.private.security.mutable-state-flags:4 string WebContentProcessLaunched
-    plistbuddy Add :com.apple.private.security.mutable-state-flags:5 string DisableLogging
```

Comment:
It looks like you also have a drive-by-fix to remove the 'DisableLogging'. Should you mention that in the ChangeLog?

Reply:
Good point, I will fix that.